packages/ui/src/command.tsx

```diff
@@ -17,20 +17,48 @@ const Command = React.forwardRef<
 	<CommandPrimitive
 		ref={ref}
 		className={cn(
-			"flex h-full w-full flex-col overflow-hidden rounded-md bg-white text-gray-950",
+			"flex h-full w-full flex-col overflow-hidden bg-white text-gray-950",
 			className
 		)}
 		{...props}
 	/>
 ));
 Command.displayName = CommandPrimitive.displayName;
 
-interface CommandDialogProps extends DialogProps {}
-
-const CommandDialog = ({ children, ...props }: CommandDialogProps) => {
+interface CommandDialogProps extends DialogProps {
+	/**
+	 * title is used for accessibility
+	 */
+	title: string;
+	/**
+	 * description is used for accessibility
+	 */
+	description: string;
+	/**
+	 * @default false
+	 */
+	showTitle?: boolean;
+	/**
+	 * @default false
+	 */
+	showDescription?: boolean;
+}
+
+const CommandDialog = ({
+	children,
+	title,
+	description,
+	showTitle,
+	showDescription,
+	...props
+}: CommandDialogProps) => {
 	return (
 		<Dialog {...props}>
 			<DialogContent className="overflow-hidden p-0 shadow-lg">
+				<DialogTitle className={cn(showTitle || "sr-only")}>{title}</DialogTitle>
+				<DialogDescription className={cn(showDescription || "sr-only")}>
+					{description}
+				</DialogDescription>
```

Comment:
these are necessary for accessibilty, otherwise you get a bunch of errors in the console/in dev